crates/polars-io/src/parquet/read/read_impl.rs

```diff
@@ -353,7 +354,10 @@ fn rg_to_dfs_prefiltered(
         });
 
         for (_, df) in &dfs {
```

Comment:
I wonder if the check is needed here as well if we can catch it at the source, ie chunkedarray construction.